deploy/codedeploy/frontend/scripts/validate-service.sh

```
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+for attempt in {1..24}; do
```

Shellcheck (0.11.0)

[warning] attempt appears unused. Verify use (or export if used externally).

(SC2034)